release/models/system/openconfig-system-bootz.yang

```diff
@@ -0,0 +1,99 @@
+module openconfig-system-bootz {
+  yang-version "1";
+
+  namespace "http://openconfig.net/yang/system-bootz";
+  prefix "oc-sys-bootz";
+
+  import openconfig-extensions { prefix oc-ext; }
+  import openconfig-system { prefix oc-sys; }
+  import openconfig-types { prefix oc-types; }
+  import openconfig-yang-types { prefix oc-yang; }
+
+  organization
+    "OpenConfig working group";
+  contact
+    "www.openconfig.net";
+
+  description
+    "This module adds configuration and operational state relating to bootz
+    service running on a network device.";
+
+
+  oc-ext:openconfig-version "1.0.0";
+  oc-ext:catalog-organization "openconfig";
+  oc-ext:origin "openconfig";
+
+  revision "2023-06-16" {
+    description
+      "Creation of bootz state paths needed for the service.";
+    reference "1.0.0";
+  }
+
+  grouping bootz-service-structural {
+    description
+      "Structural grouping for bootz service that can be enabled on
+      the system.";
+
+    container state {
+        config false;
+        description
+        "Operational state relating to the bootz service.";
+          uses bootz-state;
+    }
+  }
+
+  grouping bootz-state {
+    description
+      "State parameters required to monitor bootz service.";
+
+    leaf checksum {
+      type string;
+      default "";
+      description
+        "The current checksum of the bootz protocol buffer.
+
+        This value should refect the current sha-512 of the bootz
+        protocol buffer message BootstrapDataSigned. The protocol
+        buffer serialization must be done by tag value for each field
```

Comment:
May be worth linking the specification here by adding:

```
 reference "https://protobuf.dev/programming-guides/encoding/";
```

since I think we're just saying that this is binary marshalled proto?